### PR TITLE
Release/v0.5.6

### PR DESCRIPTION
**Pull request by LBHABEBIC**
## [0.5.6] - 06-10-2020

- add parent div to Paragraph component

- Remove grey vertical lines in Tile component, remove margin

- Add tabIndex to Table and FilterTab elements

# What?

# Why?

# Notes

# Next steps

### CODE REVIEW

File: src/components/Table/Table.tsx [hunk 1]
```diff
@@ -164,6 +164,7 @@ export const Table = ({
             <tr
               className="govuk-table__row"
               data-test="header-row"
+              tabIndex={0}
```
Comment (crablab) [inline src/components/Table/Table.tsx:167]: Is there a reason we don't use a literal here eg. `"0"`? :smile:

File: src/components/Worktray/Worktray.test.tsx [hunk 2]
```diff
@@ -146,6 +146,7 @@ describe("when given valid columns and rows", () => {
                         className="govuk-table__row"
                         data-test="header-row"
                         role="row"
+                        tabIndex={0}
```
Comment (crablab) [inline src/components/Worktray/Worktray.test.tsx:149]: Again here :smile:

**Review by crablab — COMMENTED** [review]
Just wondering why we're using `{0}` in some places and `"0"` in others for `tabIndex`? 

My understanding is that 0 would make something tab-able, so can we not just use the literal? (unless there is something React-y I'm missing!)

File: src/components/Table/Table.tsx (hunk 1 above)
Comment (LBHABEBIC) [reply src/components/Table/Table.tsx:167]: ahh not sure... you think it should be  tableIndex = '0' ?

**Pull request closed without merging**